tests/algorithms/test_bc.py

```diff
@@ -183,7 +185,7 @@ def test_that_bc_improves_rewards(
 ):
     # GIVEN
     novice_rewards, _ = evaluation.evaluate_policy(
-        cartpole_bc_trainer.policy,
+        cartpole_bc_trainer.policy,  # type: ignore[arg-type]
```

Comment:
Similar to with the seals upstream issue, let's add a TODO or GH issue for this.